trade-finance-manager-ui/server/controllers/helpers/deal-cancellation-enabled.helper.test.ts

```diff
@@ -0,0 +1,15 @@
+import { DEAL_SUBMISSION_TYPE, TEAM_IDS } from '@ukef/dtfs2-common';
+import { dealCancellationEnabled } from './deal-cancellation-enabled.helper';
+import { TfmSessionUser } from '../../types/tfm-session-user';
+
+const pimUser = { teams: [TEAM_IDS.PIM] } as TfmSessionUser;
+
+describe('dealCancellationEnabled', () => {
+  describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to false', () => {
```

Comment:
Even though the flag is `false` could you please add above assertions where you assert whether the user session is `PIM` or not please. :-)

Reply:
Added 👍